driver/kickoff.md

```diff
@@ -1,6 +0,0 @@
-# Creating a Driver
```

Comment:
Isn't that what's already explained in the architecture section? See https://doc.bblf.sh/architecture.html